### ISSUE TITLE
Cannot read properties of null (reading 'relatedModel')

### ISSUE THREAD

**Issue by Abdifatahz** [VemtoOrg/vemto2-issues#46]
I'm getting this error, after creating two new columns in the database.
The code runs with this error and no changes.

**Error Message:**
Cannot read properties of null (reading 'relatedModel')

**Stack:**
```
TypeError: Cannot read properties of null (reading 'relatedModel')
    at xK.getFilename (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:356:11940)
    at xK.render (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:351:1536)
    at file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:356:13462
    at Array.forEach (<anonymous>)
    at X9e.start (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:356:13428)
    at async oP.runGeneration (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:356:14643)
    at async oP.run (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:356:14156)
    at async C (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:384:5348)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.5
MAC M1 Sillicon
darwin 23.4.0
Locale: en-US
App version: 2.0.5

**Comment by TiagoSilvaPereira** [comment]
@Abdifatahz could you please add some details on how can I reproduce this error?

**Comment by Abdifatahz** [comment]
I think this error is related to Filament Resource when I investigated a little bit.

Suppose a renamed a database table column typo -  when I confirm the schema changes -  it passes and message shows success.

But when I click generate code  -  it shows the above error "Cannot read properties of null (reading 'relatedModel')".
When I see git code changes i see that StoriesRelashipManager is deleted as this image below show.

![Screenshot 2024-03-22 at 1 45 36 PM](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/9794917/3a17bd65-ae1c-4591-bf22-f18fe2e5b40d)

It generated Schema Changes but when I try to change filament Resource it didn't because of that "relatedModel" error.

**Comment by TiagoSilvaPereira** [comment]
Thanks @Abdifatahz I'll check this

**Comment by Abdifatahz** [comment]
@TiagoSilvaPereira thanks for checking out.

Update:

I don't know how technically can be fixed, but when I delete All has many detail child relationship on the Vemto Application tab -  everything worked perfect.

_I have just done one more step -  to recreate child relationship again using  relationship section on vemto inputs tab to avoid filament frontend error._

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.11
